scripts/run_docker_build.sh

```diff
@@ -40,7 +40,7 @@ conda install --yes anaconda-client obvious-ci
 conda install --yes conda-build-all
 
 # Workaround latest conda-build.
-conda install --yes conda-build=1.18.2
+# conda install --yes conda-build=1.18.2
```

Comment:
@msarahan [looks](https://github.com/conda-forge/staged-recipes/blob/master/scripts/run_docker_build.sh#L38-L52) like you need to update your fork. We unpinned some time ago. See https://github.com/conda-forge/staged-recipes/commit/b174fbeb81c0ea38fdb66ca5bba42bb0f66ade7f#diff-7ccd90e90efde66452169c83ae6c1930